tldraw/apps/tldraw-logseq/src/styles.css

```diff
@@ -641,7 +642,7 @@ button.tl-select-input-trigger {
   user-select: none;
 
   &[data-focused='true'] {
-    background-color: var(--ls-menu-hover-color, #f4f5f7);
+    @apply hover:bg-secondary/70 hover:text-secondary-foreground active:opacity-80;
```

Comment:
Hi, @YU000jp Appreciate for your works!  It's fixed in `feat/db` https://github.com/logseq/logseq/commit/5d1c14d760a92b0ad09db1007670fc0577ea1947

Reply:
I see.